package.json

```diff
@@ -36,5 +36,8 @@
     "eslint": ">=3",
     "eslint-plugin-jsdoc": "*",
     "precommit-hook": "3.0.0"
+  },
+  "publishConfig": {
+    "registry": "https://npm-private.atlassian.io"
```

Comment:
Why is this being added? I'm not against it, mind you, I just don't understand why it's being added because I don't have much context to the build/release flow.